jena-project/pom.xml

```diff
@@ -813,7 +833,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-report-plugin</artifactId>
-        <version>2.18.1</version>
+        <version>2.20.1</version>
         <configuration>
```

Comment:
From the parent? We may have had to set this to override in the past but that has moved on.